### ISSUE TITLE
Feedback on cosmos hub-3 to hub-4 upgrade instructions

### ISSUE THREAD

**Issue by cfl0ws** [cosmos/gaia#610]
Thanks for starting on [the upgrade instructions document](https://github.com/cosmos/gaia/blob/main/docs/migration/cosmoshub-3.md).

It appears the commit hash in the doc -

```89cf7e6fc166eaabf47ad2755c443d455feda02e```

Is different than the one listed in [Prop #37](https://cosmos.bigdipper.live/proposals/37) -

```a279d091c6f66f8a91c87943139ebaecdd84f689```

Which hash is correct?

**Comment by shahankhatch** [comment]
[Gaia v2.0.15](https://github.com/cosmos/gaia/releases/tag/v2.0.15) has a hash that corresponds with `89c...`.
[Gaia v4.0.0](https://github.com/cosmos/gaia/releases/tag/v4.0.0) has a hash that corresponds with `a27...`.
The migration is from v2.0.15 to v4.0.0.

**Comment by cfl0ws** [comment]
Thanks for clarifying, the commit hash for v4.0.0 in the doc is ```2bb04266266586468271c4ab322367acbf41188f``` -


```bash
    $ gaiad version --long
    name: gaia
    server_name: gaiad
    version: 4.0.0
    commit: 2bb04266266586468271c4ab322367acbf41188f
    build_tags: netgo,ledger
    go: go version go1.15 darwin/amd64
    build_deps:
    ...
   ```
    The version/commit hash of Gaia v4.0.0: 2bb04266266586468271c4ab322367acbf41188f

Probably needs to be changed?

**Comment by shahankhatch** [comment]
Gaia v4.0.1 will be released v soon, and will update to that tag's hash. Thanks @chris-remus

**Comment by cfl0ws** [comment]
So the version for Thursday's scheduled upgrade will be v4.0.1?

**Comment by shahankhatch** [comment]
V4.0.1 will be released as soon as v0.41.1 of the cosmos-sdk is released. Upgrading to v4.0.0 will be safe and there won't be any issues if v4.0.1 is released/installed after the Stargate Cosmos Hub upgrade; afaik the release is expected to have more robust performance in some edge cases.

**Comment by shahankhatch** [comment]
Will update the hash in the doc soon based on whatever the latest version happens to be.
